package.json

```diff
@@ -100,6 +100,7 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
+    "@emotion/css": "^11.11.2",
```

Comment:
I don't aim to keep this around for long, but the icon styling was really bothering me and I'd rather not have an entire CSS stack added to typescript... This is a pragmatic solution for now :grimacing: